### DISCUSSION TITLE
Cambio de Equipo

### DISCUSSION THREAD

**Discussion by BryanZuleta** [rramosp/ai4eng.v1#178] [UDEA Students]
Buenas tardes, quería preguntar si es posible hacer cambio de equipo ahora mismo? resulta que hicimos entrega de la primera parte del proyecto y luego de eso han cancelado la materia y me han dejado solo en el barco, un compañero me ha dicho que puedo unirme a su grupo en donde solo hay dos personas, entonces el cambio se puede hacer posible o no? teniendo en cuenta que la primera entrega solo constaba en escoger el dataset y poco más.

**Comment by Felipe-RA** [comment]
Saludos Bryan

Pasate por alguna de las sesiones sincrónicas y lo hablamos
